.changeset/polite-clocks-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+"svelte": patch
+---
+
+feat: make $derived property accessing fine-grain reactive
```

Comment:
```suggestion
feat: make effect invalidation fine-grained when accessing properties of `$derived` state
```

Reply:
It's not just effects, but other `derived`s too.